package.json

```diff
@@ -19,6 +19,7 @@
     "test:cov": "jest --coverage"
   },
   "dependencies": {
+    "@aws-sdk/client-glue": "^3.170.0",
```

Comment:
We can't do variable version installs.  I'd also keep this at the same version as the other aws-sdk packages.

Reply:
Will amend to 3.154.00